packages/kununu-utils/kununu-helpers/cookies/cookies.spec.js

```diff
@@ -18,12 +18,11 @@ describe('cookies', () => {
     expect(cookies.get('kununu_country')).toEqual('de');
   });
 
-  it('can parse json cookie', () => {
```

Comment:
Objects are handled on our side now (with JSON.stringify), this means parsed objects won't be accepted anymore. Given these conditions, this test makes no sense.

Agree?